### ISSUE TITLE
Persisting  the Solana command on Linux (Ubuntu)

### ISSUE THREAD

**Issue by litmus-zhang** [solana-developers/solana-cookbook#525]
After downloading and extracting solana-cli binaries on my machine (Ubuntu 20.04 and 22.04), when i open a new terminal, i cannot access the solana command.

**Comment by litmus-zhang** [comment]
@litmus-zhang  working on this

**Comment by Sarvesh-Ghildiyal** [comment]
Hello there, 
Did you added the path to your binaries to your system path variables?

`export PATH=$PATH:/path/to/solana/bin
`
change the "/path/to/solana/bin" to the actual path of your folder.

and then reload the shell to implement the changes made via

`source ~/.bashrc
`

**Comment by litmus-zhang** [comment]
Yeah I did,I tried it on Ubuntu 20.04 and 22,same issue after rebooting
the whole PC.

**Comment by Sarvesh-Ghildiyal** [comment]
when you add the environment variable in the particular session then in that session of terminal are you able to use the cli?

**Comment by litmus-zhang** [comment]
Unfortunately, no.
That's why I created the issue.

**Comment by Sarvesh-Ghildiyal** [comment]
i dont get it then, like you are saying, if you added the **path to environment variable** then atleast it should  atleast work on the same terminal!

can you please provide some more details, error logs or so?

**Comment by litmus-zhang** [comment]
After setting the environment variable in the terminal window, It works on
that window, but when I open another window,
It says "Solana command not found"

**Comment by Sarvesh-Ghildiyal** [comment]
Oh great! We on right track then glad 😌 it happened.

Soooo now what u gotta do is simple, just save the env variable manually open your bashrc in nano and paste the env path at the bottom of the file and save it.

`nano ~/.bashrc
`

Then, at the bottom of file paste 

`export PATH=/your/directory/path:$PATH
`

With the actual path, save it and close it.

**Comment by Johnsonojo** [comment]
Hello th

> Oh great! We on right track then glad 😌 it happened.
> 
> Soooo now what u gotta do is simple, just save the env variable manually open your bashrc in nano and paste the env path at the bottom of the file and save it.
> 
> `nano ~/.bashrc `
> 
> Then, at the bottom of file paste
> 
> `export PATH=/your/directory/path:$PATH `
> 
> With the actual path, save it and close it.

Hi @Sarvesh-Ghildiyal I'm also having the same issue. I've tried your suggestions but it still doesn't work

**Comment by willian-uiu** [comment]
when you use the command 
`export PATH="/home/userName/.local/share/solana/install/active_release/bin:$PATH"`
The SOLANA CLI turn open temporaly. You have to add this command on your .bashrc for work what you want

after do this, work on normally

link of reference:
https://phoenixnap.com/kb/linux-add-to-path
